paperqa/readers.py

```diff
@@ -48,7 +48,7 @@ def chunk_pdf(
 
     if not parsed_text.content:
         raise ImpossibleParsingError(
-            "No text was parsed from the document: either empty or corrupted."
+            f"No text was parsed from the document {doc}, either empty or corrupted."
```

Comment:
I think the repr for doc can be quite large, we should limit this

Reply:
Good point, I went with just `doc.docname`